pkg/icingadb/v1/history/comment.go

```diff
@@ -109,14 +111,58 @@ func (et CommentEventTime) Value() (driver.Value, error) {
 	}
 }
 
+// CommentEntryType represents the type of a comment in the history.
+//
+// Starting with Icinga 2 2.15, the comment entry type will always be written to Redis as a string.
+// For compatibility reasons with older history entries within Redis, it supports also uint8 values.
+type CommentEntryType struct {
+	Entry any `db:"-"`
+}
```

Comment:
Couldn't this simply be `type CommentEntryType string`? It's set to a string in all cases below already, I'm just not sure if there's still another reason for the struct around it.